server/models/ClubProfileModel.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const clubProfileSchema = new Schema({
+    name: { type: String, required: true },
+    description: { type: String, required: true },
+    lastUpdated: { type: Date, default: Date.now() },
+    imageUrl: { type: String, default: '' },
+    memberRange: { type: String, required: true },
+    acceptingMembers: { type: String, required: true },
```

Comment:
consider adding two boolean fields `springRecruiting` and `fallRecruiting` so that filtering later will be easier (can simply filter by boolean value instead of having to parse the strings)